README.md

```diff
@@ -64,3 +64,4 @@ For making any changes (from your local system), just following the following st
 Respect fellow developers, everyone might be at a different stage of skill levels 
  Encourage fellow developers, pass on an opportunity to them, if they've asked something simple, just point them to the right direction in as polite a manner as possible 
  Thank fellow developers if they help you out in any way 
+
```

Comment:
Remove this change

Reply:
modified all the changes please review again